### PR TITLE
Added generate kubectl token page

### PR DESCRIPTION
**Pull request by alexchrisramos**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-78927

## Description of changes being made

Part of epic https://jira.d2iq.com/browse/D2IQ-84949
[DOC] Feature Parity Work for Mediums and Low priority topics

Added the generate kubectl token page missing for 2.x that exist in 1.8 docs.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4581.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/security/generate-token/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+excerpt: How to generate a client access token
+beta: false
+enterprise: false
+---
+
+Obtain a token from the Kommander dashboard, on the user profile dropdown menu.
+
+![Kommander-dashboard](./kommander-generate-token.png)
+
+1.  Select the “Generate Token” button.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.1/security/generate-token/index.md:15]:
```suggestion
1.  Select the **Generate Token** button, from the user profile dropdown in the upper right corner.
```

File: pages/dkp/kommander/2.1/security/generate-token/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+title: Generate a Client Access Token
+menuWeight: 40
+excerpt: How to generate a client access token
+beta: false
+enterprise: false
+---
+
+Obtain a token from the Kommander dashboard, on the user profile dropdown menu.
+
+![Kommander-dashboard](./kommander-generate-token.png)
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.1/security/generate-token/index.md:13]: Maybe we should put this image after step 1.  Select the **Generate Token** button?

File: pages/dkp/kommander/2.1/security/generate-token/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+1.  Select the “Generate Token” button.
+
+1.  Select an identity provider and cluster. If required, you will be prompted to login again.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.1/security/generate-token/index.md:17]:
```suggestion
1.  Select an identity provider and cluster. If required, you will be prompted to log in again.
```

File: pages/dkp/kommander/2.1/security/generate-token/index.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+1.  Once logged-in, a page will show the commands required to configure kubectl to access the Konvoy cluster.
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.1/security/generate-token/index.md:19]:
```suggestion
1.  A page shows the commands required to configure `kubectl` to access your cluster.
```

File: pages/dkp/kommander/2.1/security/generate-token/index.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+<p class="message--note"><strong>NOTE:</strong> When the token expires, it is necessary to repeat the above process to obtain a fresh token. When refreshing a token, only the <code>kubectl config set-credentials</code> command needs to be executed with the new token.</p>
```
Comment (gvasquezvargas) [inline pages/dkp/kommander/2.1/security/generate-token/index.md:21]:
```suggestion
<p class="message--note"><strong>NOTE:</strong> When the token expires, you have to repeat the previous process to obtain a new token. When refreshing a token, only the <code>kubectl config set-credentials</code> command has to be executed with the new token.</p>
```

**Review by gvasquezvargas — DISMISSED** [review]
Hi, Alex! Thank you for this topic!! I made a few comments and suggestions in the 2.1 file, but these should go into all versions. Let me know if you have questions!

**Comment by alexchrisramos** [comment]
I have applied the requested changes into 2.1, 2.2 and 2.3.
Thank you!

**Comment by gvasquezvargas** [comment]
Hey @alexchrisramos , I just noticed the [package-lock.json](https://github.com/mesosphere/dcos-docs-site/pull/4581/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) file. Was that also intended to be part of the PR?

**Comment by cmastr** [comment]
@alexchrisramos we have multiple issues with this PR. there's a json file that can't be in there. And there are files that are not named index.md. so we can't merge this as-is.

**Comment by cmastr** [comment]
> @alexchrisramos we have multiple issues with this PR. there's a json file that can't be in there. And there are files that are not named index.md. so we can't merge this as-is.

Ok, that's not true. The image files are fine. just the json file.

**Comment by msdolbey** [comment]
I can take care of this json file - probably just a misfire when adding files to the commit from build or something.

**Pull request closed without merging**